docs/src/modules/components/AppSearch.js

```diff
@@ -242,18 +246,16 @@ export default function AppSearch() {
 
   return (
     <React.Fragment>
-      <SearchButton ref={searchButtonRef} onClick={onOpen}>
+      <SearchButton disableFocusRipple ref={searchButtonRef} onClick={onOpen}>
```

Comment:
@siriwatknp This creates a warning:

<img width="544" alt="Screenshot 2021-10-31 at 01 53 47" src="https://user-images.githubusercontent.com/3165635/139561462-3281331e-c6fb-498b-af85-8860269dd5f4.png">

Reply:
will create a fix.